composer.json

```diff
@@ -92,34 +92,34 @@
     ],
 
     "require": {
-        "php":                            "^5.6|^7.0",
-        "symfony/framework-bundle":       "^2.8|^3.0|^4.0",
-        "symfony/security-bundle":        "^2.8|^3.0|^4.0",
-        "symfony/options-resolver":       "^2.8|^3.0|^4.0",
-        "symfony/form":                   "^2.8|^3.0|^4.0",
-        "symfony/yaml":                   "^2.8|^3.0|^4.0",
-        "symfony/templating":             "^2.8|^3.0|^4.0",
+        "php":                            "^7.1",
+        "symfony/framework-bundle":       "^2.8 || ^3.4 || ^4.1",
+        "symfony/security-bundle":        "^2.8 || ^3.4 || ^4.1",
+        "symfony/options-resolver":       "^2.8 || ^3.4 || ^4.1",
+        "symfony/form":                   "^2.8 || ^3.4 || ^4.1",
+        "symfony/yaml":                   "^2.8 || ^3.4 || ^4.1",
+        "symfony/templating":             "^2.8 || ^3.4 || ^4.1",
 
         "psr/http-message":               "^1.0",
         "php-http/client-implementation": "^1.0",
-        "php-http/httplug":               "^1.0",
-        "php-http/client-common":         "^1.3",
+        "php-http/httplug":               "^1.1 || ^2.0",
+        "php-http/client-common":         "^1.8",
         "php-http/message-factory":       "^1.0",
-        "php-http/discovery":             "^1.0"
+        "php-http/discovery":             "^1.5"
     },
 
     "require-dev": {
-        "doctrine/orm":                 "^2.3",
-        "symfony/property-access":      "^2.8|^3.0|^4.0",
-        "symfony/validator":            "^2.8|^3.0|^4.0",
-        "symfony/twig-bundle":          "^2.8|^3.0|^4.0",
-        "symfony/stopwatch":            "^2.8|^3.0|^4.0",
-        "symfony/phpunit-bridge":       "^2.8|^3.0|^4.0",
-        "friendsofsymfony/user-bundle": "^1.3|^2.0",
-        "php-http/httplug-bundle":      "^1.7",
-        "php-http/guzzle6-adapter":     "^1.1",
-        "phpunit/phpunit":              "^5.7",
-        "friendsofphp/php-cs-fixer":    "^2.0"
+        "doctrine/orm":                 "^2.6",
+        "symfony/property-access":      "^2.8 || ^3.4 || ^4.1",
+        "symfony/validator":            "^2.8 || ^3.4 || ^4.1",
+        "symfony/twig-bundle":          "^2.8 || ^3.4 || ^4.1",
+        "symfony/stopwatch":            "^2.8 || ^3.4 || ^4.1",
+        "symfony/phpunit-bridge":       "^2.8 || ^3.4 || ^4.1",
+        "friendsofsymfony/user-bundle": "^2.0",
+        "php-http/httplug-bundle":      "^1.13",
+        "php-http/guzzle6-adapter":     "^1.1 || ^2.0",
+        "phpunit/phpunit":              "^6.5 || ^7.5",
+        "friendsofphp/php-cs-fixer":    "^2.13"
```

Comment:
This version of PHP CS Fixer doesn't support Symfony 2.8 anymore. We must either decrease the dependency, or remove Symfony 2.8 support.

Reply:
That's my plan 😄